opentracing-cdi/src/main/java/io/opentracing/contrib/cdi/internal/TracerInitializer.java

```diff
@@ -0,0 +1,86 @@
+package io.opentracing.contrib.cdi.internal;
+
+import io.opentracing.Tracer;
+import io.opentracing.contrib.tracerresolver.TracerResolver;
+import io.opentracing.util.GlobalTracer;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.context.Initialized;
+import javax.enterprise.event.Observes;
+import javax.enterprise.inject.Instance;
+import javax.inject.Inject;
+import java.util.logging.Logger;
+
+/**
+ * Registers a {@link Tracer} with the {@link GlobalTracer}, should it be a tracer that is defined by the target
+ * application. If there are no application-specific {@link Tracer} available, this initializer attempts to locate one
```

Comment:
>  aplication-specific

Could you clarify? Is it tracer from globalTraer or CDI? Simple comment about the priority would be good.